packages/devextreme/testing/tests/DevExpress.core/utils.license.tests.js

```diff
@@ -0,0 +1,97 @@
+import config from 'core/config';
```

Comment:
Let's move this test case to ```jest```

It's not cool to have one part of tests in ```jest``` and another in ```QUnit```
(Discussed with @ksercs )

Reply:
done